iceberg-rest-server/src/test/java/com/datastrato/gravitino/iceberg/integration/test/util/IcebergRESTServerManager.java

```diff
@@ -0,0 +1,167 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package com.datastrato.gravitino.iceberg.integration.test.util;
+
+import static com.google.common.util.concurrent.Uninterruptibles.sleepUninterruptibly;
+
+import com.datastrato.gravitino.Config;
+import com.datastrato.gravitino.iceberg.common.IcebergConfig;
+import com.datastrato.gravitino.integration.test.util.ITUtils;
+import com.datastrato.gravitino.rest.RESTUtils;
+import com.datastrato.gravitino.server.IcebergRESTServer;
+import com.datastrato.gravitino.server.ServerConfig;
+import com.datastrato.gravitino.server.web.JettyServerConfig;
+import com.github.dockerjava.zerodep.shaded.org.apache.hc.client5.http.classic.methods.HttpGet;
+import com.github.dockerjava.zerodep.shaded.org.apache.hc.client5.http.impl.classic.CloseableHttpClient;
+import com.github.dockerjava.zerodep.shaded.org.apache.hc.client5.http.impl.classic.HttpClients;
+import com.github.dockerjava.zerodep.shaded.org.apache.hc.core5.http.ClassicHttpResponse;
+import java.io.IOException;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Optional;
+import java.util.Properties;
+import java.util.concurrent.Future;
+import java.util.concurrent.TimeUnit;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public abstract class IcebergRESTServerManager {
+
+  protected static final Logger LOG = LoggerFactory.getLogger(IcebergRESTServerManager.class);
+
+  protected Map<String, String> customConfigs = new HashMap<>();
+  protected Config serverConfig;
+  protected String checkUri;
+
+  public abstract Path getConfigDir();
+
+  public abstract Optional<Future> doStartIcebergRESTServer() throws Exception;
+
+  public abstract void doStopIcebergRESTServer();
+
+  public static IcebergRESTServerManager create() {
+    String testMode =
+        System.getProperty(com.datastrato.gravitino.integration.test.util.ITUtils.TEST_MODE);
+    if (com.datastrato.gravitino.integration.test.util.ITUtils.EMBEDDED_TEST_MODE.equals(
```

Comment:
Please shorten the class name `com.datastrato.gravitino.integration.test.util.ITUtils`

Reply:
done